pdns/dnsdistdist/dnsdist-settings-definitions.yml

```diff
@@ -0,0 +1,1705 @@
+---
+global:
+  skip-serde: true
+  parameters:
+    - name: "acl"
+      type: "Vec<String>"
+      default: "127.0.0.0/8, 10.0.0.0/8, 100.64.0.0/10, 169.254.0.0/16, 192.168.0.0/16, 172.16.0.0/12, ::1/128, fc00::/7, fe80::/10"
+      description: "CIDR netmasks of the clients allowed to send DNS queries"
+    - name: "backends"
+      type: "Vec<BackendConfiguration>"
```

Comment:
Type is Vec<...> but default is `"true"`, is there a special meaning for that? There are quite a few instances of that.

Reply:
Yes, it's briefly documented in `dnsdist-settings-generator.py`: 

> Default values are written as quoted strings, with the exception of the
> special unquoted true value which means to use the default value for the
> object type, which needs to exist.

Reply:
Ah, right, thanks.